lib/web/app/redirect.go

```diff
@@ -78,8 +78,12 @@ const js = `
         }).then(response => {
           if (response.ok) {
             try {
-              var redirectUrl = new URL(path, url.origin)
-              window.location.replace(redirectUrl.toString());
+              if (path) {
+                var redirectUrl = new URL(path, url.origin)
+                window.location.replace(redirectUrl.toString());
+              } else {
+                window.location.replace(url.origin);
+              }
             } catch (error) {
                 // in case of malformed url, return to origin
                 window.location.replace(url.origin)
```

Comment:
for some reason GH won't let me comment there, but the comment 2 lines below should probably move up and maybe needs an update

Reply:
this comment is explaining what error would be caught if trying to `window.location.replace`, which is pretty much just a malformed url error,
but you are correct, I do think we need another comment above as well

Reply:
The one that says `// redirect to the target path and remove current URL from history (back button)`